fili-core/src/main/java/com/yahoo/bard/webservice/web/ApiRequest.java

```diff
@@ -525,7 +142,7 @@ protected static void validateTimeAlignment(
      * contains a 'startsWith' or 'contains' operation while the BardFeatureFlag.DATA_STARTS_WITH_CONTAINS_ENABLED is
      * off.
      */
-    protected Map<Dimension, Set<ApiFilter>> generateFilters(
+    default Map<Dimension, Set<ApiFilter>> generateFilters(
```

Comment:
I don't think this is the right impulse here.  This is not a method for an interface default method.